ui/app/models/clients/new-init-activity.js

```diff
@@ -1,11 +1,11 @@
 import Model, { attr } from '@ember-data/model';
 // ARG TODO copied from before, modify for what you need
-export default Model.extend({
-  total: attr('object'),
-  byNamespace: attr('array'),
-  endTime: attr('string'),
-  startTime: attr('string'),
-  clients: attr('number'),
-  distinct_entities: attr('number'),
-  non_entity_tokens: attr('number'),
-});
+export default class NewInitActivity extends Model {
```

Comment:
It might be a good idea to add model to the class name to avoid using the same name for the adapter and serializer classes such as `NewInitActivityModel`. It could be insignificant but in the case where an error is thrown the stack trace may point to the class name so might help to see that it's the model class.

Reply:
good idea. I'll amend.